app/services/rss_reader/assembler.rb

```diff
@@ -54,7 +54,7 @@ def assemble_body_markdown
     end
 
     def get_content
-      @item.content || @item.summary || @item.description
+      @item[:content] || @item[:summary] || @item[:description]
```

Comment:
Is there going to be any case where `@item` is not present? If so we might want to use `@item&.dig(..)` with the safety operator. Not necessary but thinking ahead if that's a possible case.

Reply:
That's unlikely as `@item` is individual RSS entries generated by Feedjira gem.